tests/catch/unit/occupancy/hipOccupancyMaxPotentialBlockSize.cc

```diff
@@ -69,6 +69,20 @@ TEST_CASE("Unit_hipOccupancyMaxPotentialBlockSize_rangeValidation") {
 
 }
 
+/*
```

Comment:
These lines shouldn't be commented. These are building fine on AMD and Nvidia backends. For SPIRV you could disable this test in cmake.